tools/storage-scale-driver-snap.sh

```diff
@@ -102,14 +100,73 @@ function find_versions()
 
 }
 
+function collect_csi_pod_logs()
+{
+  ns=$1
+  cmd=$2
+  since=$3
+  previous=$4
+  csi_pod_logs=${logdir}/namespaces/${ns}/pod/
+  klog="$cmd logs --namespace $ns"
+
+  for opPodName in $($cmd get pods --no-headers --namespace "$ns" -l app.kubernetes.io/name=ibm-spectrum-scale-csi-operator | awk '{print $1}'); do
+    echo "Gather data for pod/${opPodName}"
+    for containerName in $($cmd get pods "$opPodName" --namespace "$ns" -o jsonpath="{.spec.containers[*].name}"); do
+      mkdir -p "$csi_pod_logs"/"${opPodName}"/"${containerName}"
+      if [[ $since != "" ]]
+      then
+        $klog pod/"${opPodName}" -c ${containerName} --since "$since" > "$csi_pod_logs"/"${opPodName}"/"${containerName}"/"${opPodName}"-"${containerName}".log 2>&1 || :
+      else
+        $klog pod/"${opPodName}" -c ${containerName} > "$csi_pod_logs"/"${opPodName}"/"${containerName}"/"${opPodName}"-"${containerName}".log 2>&1 || :
+      fi
+      if [[ $previous != "False" ]]
+      then
+        echo "Gather data for pod/${opPodName} --previous "
+        $klog pod/"${opPodName}" -c ${containerName} > "$csi_pod_logs"/"${opPodName}"/"${containerName}"/"${opPodName}"-"${containerName}"-previous.log 2>&1 || :
+      fi
+    done
+  done
+}
+
+
+function get_kind()
+{
+
+  ns=$1
+  cmd=$2
+
+  cluster_scoped_kinds=(storageclass clusterroles clusterrolebindings nodes pv volumeattachment csinodes )
+  namespace_kinds=(pod secret configmap daemonset pvc serviceaccount deployment events CSIScaleOperator scc)
```

Comment:
SCC is cluster scoped and present only on OCP, which CNSA already collects, should we skip collecting SCC info here?